flink-table-store-core/src/main/java/org/apache/flink/table/store/table/source/Split.java

```diff
@@ -23,5 +23,5 @@
 /** An input split for reading. */
 public interface Split extends Serializable {
 
-    long rowCount();
```

Comment:
Maybe current `rowCount` is OK for now?
`SplitStatistics` is too heavy (column stats are very heavy to compute), we can introduce it in future.
WDYT?

Reply:
Agree, I think the column stats should be optimized instead of reading from file meta directly and we can discuss about it later